src/docs/funding.md

```diff
@@ -0,0 +1,51 @@
+## Funding
+
+The Simple Standard for Sharing Ontological Mappings (SSSOM) is a community-driven project which has received support from many different sources.
+We list the most important ones in the following.
+
+### Volunteering efforts
+
+A huge fraction of the work on SSSOM has been done by volunteers without dedicated grant support.
+We hereby acknowledge their contributions as being absolutely essential. A selection of amazing contributions (by no means exhaustive):
+
+- The development of [SSSOM Java](https://incenp.org/dvlpt/sssom-java/)
+- Hundreds of careful contributions to discussions on the [SSSOM issue tracker](https://github.com/mapping-commons/sssom/issues)
+- The first draft of the [Mapping Registry Cookiecutter](https://github.com/mapping-commons/mapping-commons-cookiecutter)
+- We try to keep track of other [Community efforts here](https://github.com/mapping-commons/sssom/discussions/318)
+
+### Phenomics First (NIH / NHGRI #1RM1HG010860-01)
```

Comment:
Please note that there is currently already a “funding” section at the bottom of the [home page](https://mapping-commons.github.io/sssom/#acknowledgements). I have no opinion on whether that information should be on the home page or on a separate page, but it shouldn’t be repeated in two different pages (if only to avoid possible discrepancies between the two pages).

I’d suggest removing the entire “Acknowledgements” section on the home page. Your call on whether you want to do that as part of this PR (no objection if you want to do that in a follow-up PR).

Reply:
I just replaced the section by a link rather than removing it altogether. Does this work?

Reply:
It works for me.